src/main/java/com/qaprosoft/carina/demo/guipractice/components/ProductInCart.java

```diff
@@ -0,0 +1,16 @@
+package com.qaprosoft.carina.demo.guipractice.components;
+
+import org.openqa.selenium.WebDriver;
+
+import com.qaprosoft.carina.core.gui.AbstractUIObject;
+
+public class ProductInCart extends AbstractUIObject{
+
+	public ProductInCart(WebDriver driver) {
```

Comment:
add elements and methods that needed for your test cases

Reply:
Just for making sure if I understood. An element could be (EX) the title of a product and a method could be readTitle() (where I can use .getText or getTitle?